src/diffusers/loaders/unet.py

```diff
@@ -684,13 +684,20 @@ def _convert_ip_adapter_image_proj_to_diffusers(self, state_dict):
                 diffusers_name = key.replace("proj", "image_embeds")
                 updated_state_dict[diffusers_name] = value
 
-        elif "proj.3.weight" in state_dict:
+        elif "proj.0.weight" in state_dict:
```

Comment:
Is using this key a better option? Can we use a more resilient condition here to avoid side-effects?

Reply:
I changed to `proj.0.weight` because both IPAdapter Full and FaceID state dicts have it, while the IPAdapter Full `proj.3.weight` key is named `norm.weight` in the FaceID model